bin/pycbc_live

```diff
@@ -83,40 +98,72 @@ class LiveEventManager(object):
+        followup_ifos = [] if followup_ifos is None else followup_ifos
 
         template_id = triggers['foreground/' + ifos[0] + '/template_id']
         htilde = bank[template_id]
 
-        subthreshold_sngl_time = numpy.mean(
-                [triggers['foreground/' + ifo + '/end_time'] for ifo in ifos])
+        coinc_times = {ifo: triggers['foreground/' + ifo + '/end_time'] for ifo in ifos}
 
-        for ifo in ifos + (followup_ifos or []):
-            if ifo in ifos:
-                trig_time = triggers['foreground/' + ifo + '/end_time']
-            else:
-                trig_time = subthreshold_sngl_time
+        # Get the SNR series for the ifos that made the initial coinc
+        for ifo in ifos:
             # NOTE we only check the state/DQ of followup IFOs here.
             # IFOs producing the coincidence are assumed to also
             # produce valid SNR series.
             snr_series = compute_followup_snr_series(
-                    data_readers[ifo], htilde, trig_time,
-                    check_state=(ifo in followup_ifos))
+                    data_readers[ifo], htilde, coinc_times[ifo],
+                    check_state=False)
+
+            if snr_series is not None:
+                out[ifo] = {'snr_series': snr_series}
+
+        # Determine if the other ifos can contribute to the coincident event
+        for ifo in followup_ifos:
+            snr_series, ptime, pvalue = followup_event_significance(ifo, 
+                            data_readers[ifo], bank, template_id, coinc_times)
+
             if snr_series is not None:
                 out[ifo] = {'snr_series': snr_series}
 
+            self.combine_far_with_followup(ifos[0], ifo, triggers,
+                                           snr_series, ptime, pvalue)
         return out
 
+    def combine_far_with_followup(self, coinc_ifo, ifo, triggers, snr_series, ptime, pvalue):      
+        # Set new ifar
+        triggers['foreground/ifar'] = combine_ifar_pvalue(triggers['foreground/ifar'], pvalue)
+        
+        # Copy the triggers
+        for key in triggers.copy():
+            if 'foreground/{}/'.format(coinc_ifo) in key:
+                _, _, name = key.split('/')
+                triggers['foreground/{}/{}'.format(ifo, name)] = triggers[key]
+
+        # Set the critical stats manually
+        triggers['foreground/{}/end_time'.format(ifo)] = float(ptime)
+        triggers['foreground/{}/snr'.format(ifo)] = abs(snr_series.at_time(ptime))
+        triggers['foreground/{}/stat'.format(ifo)] = triggers['foreground/{}/snr'.format(ifo)]
+        triggers['foreground/{}/coa_phase'.format(ifo)] = numpy.angle(snr_series.at_time(ptime))
+        triggers['foreground/{}/chisq'.format(ifo)] = 0
+                     
     def check_coincs(self, ifos, coinc_results, psds, f_low,
-                     data_readers, bank, followup_ifos=None):
+                     data_readers, bank):
         """ Save zerolag triggers to a coinc xml file """
```

Comment:
is this all the function does? looks like it also manages the followup, then say so